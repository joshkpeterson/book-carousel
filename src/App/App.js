import "./App.scss"
import Carousel from "./components/Carousel"
import carouselData from "../data/carousel"
import React from "react"

export default function App() {
  return (
    <main className="app">
      Welcome to Scribd!
      <Carousel {...carouselData} />
    </main>
  )
}
