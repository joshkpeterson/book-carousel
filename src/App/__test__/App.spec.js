import App from "../App"
import React from "react"
import renderer from "react-test-renderer"

describe("<App />", () => {
  it("matches the snapshot", () => {
    const component = renderer.create(<App />)
    expect(component).toMatchSnapshot()
  })
})
