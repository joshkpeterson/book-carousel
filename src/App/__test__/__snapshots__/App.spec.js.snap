// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the snapshot 1`] = `
<main
  className="app"
>
  Welcome to Scribd!
  <section
    className="carousel"
  />
</main>
`;
